app/components/hero/Hero.module.css

```diff
@@ -1,19 +1,179 @@
 .container {
   position: relative;
-  z-index: 2;
+  z-index: 12;
   width: 100%;
-  max-width: 1920px;
-  height: 100vh;
+  min-width: 320px;
+  max-width: 1728px;
+  height: 100svh;
+  min-height: 700px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bottomContainer {
+  position: absolute;
+  grid-area: 7;
+  top: 10px;
+  width: 70vw;
+  max-width: 510px;
+  min-height: calc(100svh - 200px);
+  max-height: 980px;
+  display: flex;
```

Comment:
vzhledem k tomu, ze vsechny decka mas v `position:absolute`, tak tohle nic nedela. Ten `flex-direction` a `justify-content` je tam taky na nic

Reply:
to je pro mobilni a tablet verzi. tam to funguje tak jak jsem puvodne zamyslel a to tak, ze se velikost hero prizpusobuje vysce okna prohlizece. tudiz kdyz budes mit mobil/tablet s hodne vysokym displayem, tak neuvidis how it works sekci.